processor/src/test/java/org/mapstruct/ap/test/bugs/_1685/UserMapper.java

```diff
@@ -38,7 +38,6 @@
 
     @InheritInverseConfiguration
     @BeanMapping( nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.SET_TO_DEFAULT )
-    @Mapping( target = "phone", source = "contactDataDTO.phone", defaultValue = "0" )
```

Comment:
Shouldn't we just remove the `defaultValue`? At least it was like that before.

Reply:
it's inherited.. To reassure you: the fixture did not change :smile:

Reply:
Ah yes I missed thw inheritance 😀